packages/orchestrator-ui-components/src/components/WfoProcessList/WfoProcessesList.tsx

```diff
@@ -241,34 +218,24 @@ export const WfoProcessesList = ({
     return (
         <WfoTableWithFilter<ProcessListItem>
             queryString={queryString}
-            data={
-                data ? mapGraphQlProcessListResultToProcessListItems(data) : []
-            }
+            data={mapGraphQlProcessListResultToProcessListItems(
+                processes || [],
+            )}
             tableColumns={mapSortableAndFilterableValuesToTableColumnConfig(
                 tableColumns,
                 sortFields,
                 filterFields,
             )}
             dataSorting={dataSorting}
             pagination={pagination}
-            isLoading={isLoading}
+            isLoading={isFetching}
             hasError={isError}
             defaultHiddenColumns={defaultHiddenColumns}
             localStorageKey={localStorageKey}
             detailModalTitle={'Details - Process'}
             onUpdateQueryString={getQueryStringHandler(setDataDisplayParam)}
             onUpdatePage={getPageChangeHandler(setDataDisplayParam)}
             onUpdateDataSort={getDataSortHandler(setDataDisplayParam)}
-            onExportData={csvDownloadHandler(
-                getProcessListForExport,
-                mapGraphQlProcessListResultToProcessListItems,
-                mapGraphQlProcessListResultToPageInfo,
-                Object.keys(tableColumns),
-                getCsvFileNameWithDate('Processes'),
-                addToast,
-                tError,
-            )}
-            exportDataIsLoading={isFetchingCsv}
```

Comment:
I think removing a feature is not a good idea. I would expect that both data fetching ways can/should work together on one page.